test/e2e/pd.go

```diff
@@ -173,7 +173,7 @@ var _ = Describe("Pod Disks", func() {
 })
 
 func createPD() (string, error) {
-	if testContext.Provider == "gce" {
```

Comment:
I know this isn't technically part of this PR, but could I get you to put a log statement above line 169 above, right above the `break`? In viewing the test output to see whether it truly passed, it's really confusing to have "error, ... , error, ... , error, ..., exit status 1 TEST PASSED!". I would hope we could add:

```go
Logf("Successfully deleted PD %q", diskName)
```

I affectionally call this affirmative logging, and it hugely helps test output readability IMHO.